frontend/src/app/admin/settingsModal/tokenConfirmation/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import Modal from "@/components/modal/modal";
+import { useRouter } from "next/navigation";
+import Confirmation from "@/components/tokenConfirmation/tokenConfirmation";
+
+export default function TokenConfirmation() {
+  const router = useRouter();
+
+  const handleModalClose = () => {
+    router.back();
+  };
+
+  return (
+    <Modal
+      closeModal={handleModalClose}
+      title="Settings"
+      content={Confirmation()}
```

Comment:
Is "Settings" the correct title? Also I don't think the brackets are necessary

Reply:
Changed the title to "Warning". The brackets on Confirmation are necessary because content expects an `Element`, and `Confirmation` is `() => Element`.